CHANGELOG.md

```diff
@@ -39,13 +39,34 @@ Bottom level categories:
 
 ## Unreleased
 
+## 22.0.0 (2024-07-17)
+
+### Overview
+
+### Our first major version release!
+
+For the first time ever, WGPU is being released with a major version (i.e., 22.* instead of 0.22.*)! Maintainership has decided to fully adhere to [Semantic Versioning](https://semver.org)'s recommendations for versioning production software. According to [SemVer 2.0.0's own Q&A about when to use 1.0.0 and beyond](https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100):
```

Comment:
suggestion:

```suggestion
For the first time ever, WGPU is being released with a major version (i.e., 22.* instead of 0.22.*)! Maintainership has decided to fully adhere to [Semantic Versioning](https://semver.org/)'s recommendations for versioning production software. According to [SemVer 2.0.0's Q&A about when to use 1.0.0 versions (and beyond)](https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100):
```